app/controllers/v0/form1010_ezr_attachments_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module V0
+  class Form1010EzrAttachmentsController < ApplicationController
+    include FormAttachmentCreate
+    service_tag 'health-information-update'
+
+    FORM_ATTACHMENT_MODEL = Form1010EzrAttachment
+
+    def create
+      if Flipper.enabled?(:form1010_ezr_attachments_controller)
+        super
+      else
+        raise Common::Exceptions::InternalServerError, ArgumentError.new(
+          "The 'create' route for V0::Form1010EzrAttachmentsController is currently unavailable"
+        )
+      end
```

Comment:
Added a flipper so we can test on the frontend after this code is merged.

Reply:
Do we need this flipper toggle on the backend? Since it's a new route anyways couldn't we just have the toggle on the front end for what endpoint gets sent the attachment? Technically someone could send a request to it if they knew it existed, but that seems unlikely plus it would probably fail validation. That could simplify this PR a bit too.

Reply:
Well, from what I see, we'll need the flipper on the backend and frontend. I think you need a pair.

Reply:
I went ahead and removed the feature toggle. I updated the tests to reflect the changes.